src/features/components/molecules/language-switcher/index.tsx

```diff
@@ -25,6 +25,8 @@ const LanguageSwitcher = () => {
             </DropdownMenu.Trigger>
 
             <DropdownMenu.Content
+                //It's due to radix default z-index: auto;
+                style={{ zIndex: 9 }}
```

Comment:
could you please make it like in this commit? Soon it'll merged, so better pass z-index in updated `Box` via props
https://github.com/binary-com/deriv-com/pull/4795/commits/aa8353954b7e7e9e0b8ef644e97c9f7a0e605516

Reply:
Hi @NikitK-deriv this PR is not valid anymore as it is resolved here #4931 , I will close this PR for now 🙏  Thanks